src/pages/Detail/Detail.js

```diff
@@ -1,12 +1,111 @@
-import React from "react";
+import React from 'react';
+import { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
 import './Detail.scss';
 
 const Detail = () => {
+  const params = useParams();
+  const productId = params.id;
+  const [productDetail, setProductDetail] = useState('');
```

Comment:
state의 초기값을 올바른 형태로 주는 연습이 필요합니다!

```suggestion
  const [productDetail, setProductDetail] = useState({});
```

Reply:
수정완료